README.md

```diff
@@ -1 +1,59 @@
-# Catalog-of-my-things
\ No newline at end of file
+# catalog-of-my-things
+An console app t o add resources(`music albums`, `authors`, `genres` and `books`) to`json` files and list them
+
+## Built With
+Ruby
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Ruby
+
+### Install
+
+1. Go to the terminal and `cd` into the directory of your preference.
+2. Run `git clone git@github.com:basitali111/Catalog-of-my-things.git` to clone the app.
+3. Open the project in a code editor of your choice that is configured to use Ruby.
+4. Run `ruby main.rb` to start the app
+
+### Run tests
+1. Run `gem install rspec` to install the `RSpec testing framework`
+2. Run `rspec spec` to test all the test files at once
+3. Run `rspec spec/name_of_test_file.rb` to test a particular file
+
+## Brief overview
+
+[Google Drive Link](https://drive.google.com/file/d/1Hq9qMQqJvaoQq7XeNukZkrBAut98VVlz/view?usp=share_link)
```

Comment:
- [ ] Nice work done on your project and your video presentation was well explanatory, but as a requirement, the maximum length of the video is 5 minutes yours was 6.47 minutes, kindly reduce it.

Reply:
Hi @danielufeli, thank you so much for taking the time to leave this excellent review. Making the change and we really appreciate your input to make our code better. Cheers!